services/frontend-service/src/ui/App/index.tsx

```diff
@@ -23,6 +23,26 @@ import { PanicOverview, showSnackbarWarn, UpdateOverview, useReleaseDialog, useR
 import { useApi } from '../utils/GrpcApi';
 import { AzureAuthProvider, UpdateFrontendConfig, useAzureAuthSub } from '../utils/AzureAuthProvider';
 import { Snackbar } from '../components/snackbar/snackbar';
+import { mergeMap, retryWhen } from 'rxjs/operators';
+import { Observable, throwError, timer } from 'rxjs';
+
+// retry strategy: retries the observable subscription with randomized exponential backoff
+// source: https://www.learnrxjs.io/learn-rxjs/operators/error_handling/retrywhen#examples
+function retryStrategy(maxRetryAttempts = 3) {
```

Comment:
3 seems low to me, I would suggest 6 or more

Reply:
I also think so. With just three retries and the backoff formula you used, you would stop retrying after 17 seconds. I think you can safely go to much higher retry counts.

Reply:
yes, I think we can go much higher. Since it's exponential, it shouldn't be a problem

Reply:
Okay, I increased the attempts to 8 which takes about 2 mins to give up.